pkg/addons/openebs/static/metadata.yaml

```diff
@@ -0,0 +1,42 @@
+#
+# this file is automatically generated by buildtools. manual edits are not recommended.
+# to regenerate this file, run the following commands:
+#
+# $ make buildtools
+# $ output/bin/buildtools update addon <addon name>
+#
+version: 4.1.0
+location: oci://proxy.replicated.com/anonymous/ttl.sh/ricardox/openebs
```

Comment:
this image location uses ttl.sh, this doesn't seem expected

Reply:
These references to ttl are there for testing purposes. :-) Let me clean this up.